packages/components/scripts/cleanupLoader.ts

```diff
@@ -35,7 +39,7 @@ export const defineCustomElements = (options) => {
   globalScripts();
   return bootstrapLazy(
     // prettier-ignore
-    [${lazyData}],
+    ${JSON.parse(lazyData)},
```

Comment:
Why not capture JSON.parse as well? Currently its parameter is captured and then JSON.parse is added around it again.